assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-restricted.yaml

```diff
@@ -50,21 +50,21 @@ spec:
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: restricted-psp
+  name: zz-restricted-psp
```

Comment:
Not sure if it make sense to rename the role. I'd keep it the same as everything else and only change the PSP object name, as it has functional meaning.

Reply:
Yeah I wondering what is the right thing to do here, but eventually decided on renaming all the restricted related thing to prefix `zz-`

Reply:
removed it.